From 8bc73c99e081dfe032cea0e69195ed87dbc4ed64 Mon Sep 17 00:00:00 2001
From: Peter Vary <pvary@cloudera.com>
Date: Fri, 18 Nov 2016 10:25:28 +0100
Subject: [PATCH 0824/1164] CDH-41489: HIVE-9518: Implement MONTHS_BETWEEN
 aligned with Oracle one (Alex Pivovarov via Jason
 Dere)

q.out file fix

Change-Id: Ie5524738f1201b8321d963ea27fce79eb4f2fa5e
---
 .../clientpositive/udf_months_between.q.out        |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ql/src/test/results/clientpositive/udf_months_between.q.out b/ql/src/test/results/clientpositive/udf_months_between.q.out
index 95f756f..c8beeed 100644
--- a/ql/src/test/results/clientpositive/udf_months_between.q.out
+++ b/ql/src/test/results/clientpositive/udf_months_between.q.out
@@ -30,11 +30,11 @@ STAGE PLANS:
         TableScan
           alias: _dummy_table
           Row Limit Per Split: 1
-          Statistics: Num rows: 0 Data size: 1 Basic stats: PARTIAL Column stats: COMPLETE
+          Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
             expressions: 1.03225806 (type: double)
             outputColumnNames: _col0
-            Statistics: Num rows: 0 Data size: 1 Basic stats: PARTIAL Column stats: COMPLETE
+            Statistics: Num rows: 1 Data size: 8 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
 
 PREHOOK: query: select
-- 
1.7.9.5

